pkg/prowcopy/prowcopy.go

```diff
@@ -175,3 +208,51 @@ func getJobConfig(match string, c Config) (*prowgen.ReleaseBuildConfiguration, e
 
 	return jobConfig, nil
 }
+
+// JobConfigCopiedInjectors are configured from the base branch. They're applied to
+// generated Prow jobs for the target branch.
+type JobConfigCopiedInjectors []prowgen.JobConfigInjector
+
+func (jcis JobConfigCopiedInjectors) Inject(prowcopyCfg *Config, prowgenCfg *prowgen.Config, openShiftRelease prowgen.Repository) error {
+	for _, jci := range jcis {
+		sourceBranchName, targetBranch := prowcopyCfg.FromBranch, prowcopyCfg.Branch
+		var sourceBranch *prowgen.Branch
+		// Injectors need to be applied to the new branch in the same way as they were applied
+		// to the source branch when its config was generated.
+		for branchName, branch := range prowgenCfg.Config.Branches {
+			if branchName == sourceBranchName {
+				sourceBranch = &branch
+			}
+		}
```

Comment:
Can we just do

```golang
sb, ok := prowgenCfg.Config.Branches[sourceBranchName]
if !ok { }
sourceBranch := &sb
```

Reply:
though, it's a nit as the list is very small to not warrant the optimization

Reply:
Sorry. Will do, just a moment.

Reply:
Nevermind. Next time.